symfony/phpunit-bridge/3.3/manifest.json

```diff
@@ -5,6 +5,7 @@
         "tests/": "tests/"
     },
     "gitignore": [
+        ".phpunit",
```

Comment:
should be `/bin/.phpunit` instead

Reply:
wouldn't account for `%BIN_DIR%`, while this does